client/src/client-only-routes/ShowUser.js

```diff
@@ -14,8 +13,11 @@ import {
 } from '@freecodecamp/react-bootstrap';
 import Helmet from 'react-helmet';
 
+import { apiLocation } from '../../config/env.json';
+
 import {
   isSignedInSelector,
+  hardGoTo,
```

Comment:
```suggestion
  hardGoTo as navigate,
```
If we alias like this, we can clean up a little.  See below.